### DISCUSSION TITLE
How can I force textbox lostfocus by code?

### DISCUSSION THREAD

**Discussion by kyuranger** [AvaloniaUI/Avalonia#14489] [Q&A] [answered]
What I want to do is to lose focus when pressing the enter button in textbox.

In WPF there is a Keyboard.ClearFocus method:
[https://learn.microsoft.com/en-us/dotnet/api/system.windows.input.keyboard.clearfocus?view=windowsdesktop-8.0&redirectedfrom=MSDN#System_Windows_Input_Keyboard_ClearFocus](https://learn.microsoft.com/en-us/dotnet/api/system.windows.input.keyboard.clearfocus?view=windowsdesktop-8.0&redirectedfrom=MSDN#System_Windows_Input_Keyboard_ClearFocus)

However, this method does not exist in Avalonia.

How can I achieve this?

**Comment by thevortexcloud** [comment, marked as answer]
I believe you can use this: https://docs.avaloniaui.net/docs/concepts/services/focus-manager

**Reply by kyuranger** [reply]
It works! Thank you!
